app/experimenter/nimbus-ui/src/hooks/useCommonForm/index.test.tsx

```diff
@@ -144,5 +155,42 @@ describe("hooks/useCommonForm", () => {
         expect(screen.queryByTestId(name)).toBeInTheDocument();
       });
     });
+
+    it("FormBranches", () => {
```

Comment:
Seems like this belongs in `FormBranch.test.tsx`?

Reply:
Oh I should actually call this `FormBranch` not `FormBranches`.

I'm using this chunk of the test file (everything under the "is used on expected fields" describe) to make sure what `useCommonForm` provides, most commonly `FormErrors` and `formControlAttrs`, are called with each field name  as expected. I _could_ move the entire describe block and following grouped tests into their corresponding form tests if that's preferred, but it's also sort of nice to have similarly grouped tests in this file rather than in their own form files? I dunno, check out the test file as a whole and let me know what you think?